dex/testing/loadbot/mantle.go

```diff
@@ -361,7 +361,7 @@ func mustJSON(thing interface{}) string {
 }
 
 // Convert the conventional units value to atoms/sats.
-func toAtoms(v float64) uint64 {
+func encodeRate(v float64) uint64 {
 	return uint64(math.Round(v * 1e8))
```

Comment:
got a 1e8 here still :)
Also about 40 lines down.  If loadbot is an exception for now though, that's fine.